apps/mark-scan/frontend/vite.config.ts

```diff
@@ -72,8 +72,13 @@ export default defineConfig((env) => {
       ],
     },
 
+    // When env var is set, disable hot reloading. We support this option because VxMarkScan
+    // hardware has limited resources and hot reloading can cause poor performance during development
     server: {
-      hmr: false
+      hmr:
+        // Values are not parsed to booleans in this file
+        rootDotenvValues['REACT_APP_VX_DISABLE_MARKSCAN_HOT_RELOAD'] === 'TRUE'
+        ? false : undefined
```

Comment:
This is clunky but felt necessary since I don't think the feature flag lib is usable this early in the build process

Reply:
At this point you can just access `process.env.DISABLE_MARKSCAN_HOT_RELOAD` directly–it's just NodeJS.